transports/msmq/distributor/error-handling.md

```diff
@@ -30,7 +30,7 @@ Immediate Retries are always performed on the worker.
 
 The behavior of Delayed Retries is different between major versions.
 
-Delayed Retries has two configurable items TimeIncrease and NumberOfRetries.
+Delayed Retries has two configurable items *TimeIncrease* and *NumberOfRetries*.
```

Comment:
@SzymonPobiega can you help me understand the note in this file:

> NOTE: If settings are managed in XML configuration a comment stating that changes should be updated on the distributor and worker configuration files.

I wasn't able to grasb this